content/slice/encoding/icerpc.md

```diff
@@ -18,38 +18,146 @@ It fills the payload continuation of an outgoing response (resp. outgoing reques
+| facet                    | Ice facet computed from the service address fragment                               |
+| invocationMode           | `Twoway`                                                                           |
+| secure                   | `false`                                                                            |
+| protocolMajor            | `1` for ice, `2` for icerpc                                                        |
+| protocolMinor            | `0`                                                                                |
+| encodingMajor            | `1`                                                                                |
+| encodingMinor            | `1`                                                                                |
+| serverAddressList        | Computed from the server address and alt server address(es) of the service address |
+| adapterId                | The value of the `adapter-id` parameter                                            |
+
+The server addresses use only the `Uri` transport code when the protocol is `icerpc`.
+
+## Decoding a ServiceAddress {% icerpcSlice=true %}
+
+When the IceRPC + Slice integration decodes a Slice1-encoded [service address] into a non-null service address, it
+processes the `ServiceAddressData` fields as follows:
+
+| ServiceAddressData field | Service address decoded by IceRPC + Slice                                           |
+|--------------------------|-------------------------------------------------------------------------------------|
+| identity                 | Converted into the service address path                                             |
+| facet                    | Converted into the service address fragment                                         |
+| invocationMode           | Ignored                                                                             |
+| secure                   | Ignored                                                                             |
+| protocolMajor            | `1` for ice, `2` for icerpc; other values are invalid                               |
+| protocolMinor            | Must be 0                                                                           |
+| encodingMajor            | Ignored                                                                             |
+| encodingMinor            | Ignored                                                                             |
+| serverAddressList        | Converted into the server address and alt server address(es) of the service address |
+| adapterId                | Converted into an `adapter-id` parameter if not empty                               |
+
+The proxy options ignored and therefore lost during decoding are invocation mode (`-t`, `-o`, `-D` etc.), secure
+(`-s`), and encoding (`-e`).
 
```

Comment:
Why do we mention proxy options here?

Reply:
I don't understand your question. 

Is it the term "proxy options" that you dislike?

Reply:
This section now describes the decoding of a service address, this decoding is urelated to proxies

Reply:
You're right, I deleted this sentence. The table above already shows we ignore these fields during decoding.